sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectView.groovy

```diff
@@ -106,6 +108,37 @@ class ProjectView extends ProjectDesign {
         headerRow.getCell("SampleDataAvailable").setDescription("Number of available raw datasets.")
     }
 
+    private void addSorting(){
+        sort.setItems(["Subscribed", "Not Subscribed"])
```

Comment:
Its sorting, you could also view it as sort descending and ascending. I just checked how others do it. This is how github does it:

![Screenshot 2022-01-26 at 08 58 38](https://user-images.githubusercontent.com/20778777/151124599-7be0a901-532a-4210-82a7-0fdad0b016be.png)

Do you think I should change the naming?

Reply:
Yes. I do not see how you can display the sort order to the user. Subscribed and not subscribed are not comparisons.

Reply:
What would be more intuitive for you?

Reply:
and I am also not sure if I get what you dont understand?

Reply:
I don't think this can be done as sorting. When sorting things are compared. I do not get any comparison from `Subscribed` `Not Subscribed`. Something like `Subscribed First` or `Subscribed Last` would have an ordering thou